examples/custom-service.mdx

```diff
@@ -71,20 +157,37 @@ async fn init(
     Ok(PoolService { pool })
```

Comment:
It looks like you forgot to update this, also see below comment for what I think we should do in the main function.

Reply:
Oops 😅 this will be removed in the next commit